### PR TITLE
0.0.14

### PR DESCRIPTION
**Pull request by azazellochg**
Add shlex kwarg for parsing multi-item fields (@jamesmkrieger). False by default as it slow down parsing (e.g. iterRows) by a lot.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -63,6 +63,29 @@ Alternatively, you can use **iterRows** method which also supports sorting by a
...
+     _enabled
+     _image
+     _itemId
+     _nmaDisplacements
+     _shiftX
+     _shiftY
+     -180.009437   -96.153255   243.098777 8.055914e-04                    1 000021@Runs/000088_ProtImportParticles/extra/img.stk                   21 '    -2.733360    -2.468260    -0.090721 '    -4.306965    -0.040850
+        0.225648   206.555090     9.899219 7.811945e-04                    1 000002@Runs/000088_ProtImportParticles/extra/img.stk                    2 '   -41.891100   -41.568300     0.332499 '    -4.426523    -1.497812
+
+For such cases, you need to pass **shlex=True** to the Table, so that each string of columns is splited using shlex library. However, this requires emtable v0.0.14+
```
Comment (jamesmkrieger) [inline README.rst:87]: Split is already the past participle too. Splitted would need two Ts if it were used but I don’t think there are any cases were it is.

File: emtable/metadata.py [hunk 2]
```diff
@@ -575,6 +574,14 @@ def _getFormatStr(v):
     return '.6f' if isinstance(v, float) else ''
 
 
+def _split(string, shlex=False):
+    if shlex:
+        import shlex
```
Comment (delarosatrevin) [inline emtable/metadata.py:579]: I thought that @azazellochg commented about this. This import should be at the top level if shlex is already part of the standard library and does not introduce any problematic dependency

**Review by delarosatrevin — CHANGES_REQUESTED** [review]
Thanks @jamesmkrieger for the improvement! Is there any difference (in speed or results) when using shlex? I don't like to let the option to use it or not to the user (developer) of the emtable API. I think one should always get the correct results and not think if your table will contain quoted values. What about if we always use shlex for splitting?

**Comment by azazellochg** [comment]
> Thanks @jamesmkrieger for the improvement! Is there any difference (in speed or results) when using shlex? I don't like to let the option to use it or not to the user (developer) of the emtable API. I think one should always get the correct results and not think if your table will contain quoted values. What about if we always use shlex for splitting?

Shlex, if used by default, slows down our example tests x10 times

File: emtable/metadata.py (hunk 2 above)
Comment (azazellochg) [reply emtable/metadata.py:579]: I'll fix it

**Comment by delarosatrevin** [comment]
This is a serious slowdown....anyway might be good to be able to handle these cases without shlex.

**Comment by azazellochg** [comment]
While having quotes as delimiters for the column value is allowed by STAR format, this is the first case I see anybody using them. There are rare cases when relion has "" (empty) values but that's it

**Comment by jamesmkrieger** [comment]
We actually have another case that made me aware of it in our Zernike3D programs. There’s also another case where we’re thinking of using it.

I guess there won’t be that many others though.

**Comment by azazellochg** [comment]
Extra kwarg removed, quotes are checked internally now as requested by @delarosatrevin .

**Comment by jamesmkrieger** [comment]
Looks good to me

**Comment by azazellochg** [comment]
One more case covered, when the column value is "..." and not " ggg "

**Review by delarosatrevin — APPROVED** [review]
Thanks @jamesmkrieger and @azazellochg !

**Pull request merged**